README.md

```diff
@@ -5,35 +5,62 @@
 [![bitHound Dependencies](https://www.bithound.io/projects/badges/f907f2c0-bfbd-11e5-b2ad-73729a2991e3/dependencies.svg)](https://www.bithound.io/github/bithound/cli.bithound.io/master/dependencies/npm)
 [![bitHound Dev Dependencies](https://www.bithound.io/projects/badges/f907f2c0-bfbd-11e5-b2ad-73729a2991e3/devDependencies.svg)](https://www.bithound.io/github/bithound/cli.bithound.io/master/dependencies/npm)
 
-## Alpha Pre-release
-
-This command line tool is currently in a closed alpha preview.  If you would like to try it out get in touch with Gord at gord@bithound.io
-
 ## Installation:
 Add the latest version of `bithound` to your package.json:
+
 ```
 npm install bithound --save-dev
 ```
 
 # Usage:
+This script `bin/bithound` provides two cli commands:
+
+  - `check (git url || repo token)` Checks bitHound for failing files or dependencies.
+  
+  - `token` Opens your browser to the repo settings page of the current project.
+  
+If you include `node_modules/.bin` in your `$PATH`, you can run this cli with:
```

Comment:
You'd need to add it specifically to a script section in package.json so it doesn't work 'out of the box' so to speak where it seems that a lot of people either modify their $PATH or run it direct. I'm not sure if a 3 option of running it will be more benficial....